tests/integration/test_alembic_unit_split_cp09.py

```
@@ -0,0 +1,408 @@
+        cwd=ROOT,
+        text=True,
+        capture_output=True,
+        check=True,
+    )
+    major_version = server_version.stdout.strip().split(".", 1)[0]
+    candidate_paths = [
+        shutil.which(f"pg_dump@{major_version}"),
+        shutil.which(f"pg_dump-{major_version}"),
+        f"/opt/homebrew/opt/postgresql@{major_version}/bin/pg_dump",
+        f"/usr/local/opt/postgresql@{major_version}/bin/pg_dump",
+        f"/opt/homebrew/opt/libpq/bin/pg_dump",
```

Ruff (0.15.10)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)